tests/Microsoft.Identity.Test.Unit.net45/CacheTests/TokenCacheTests.cs

```diff
@@ -388,7 +388,20 @@ public void GetAppTokenFromCacheTest()
         [TestCategory("TokenCacheTests")]
         public void DoNotSaveRefreshTokenInAdalCacheForMsalB2CAuthorityTest()
         {
-            var serviceBundle = TestCommon.CreateDefaultServiceBundle();
+            var appConfig = new ApplicationConfiguration()
```

Comment:
nit: can you extract this AppConfig constructor to do all of the "null the things" and have a function that just sends in the different authority uri?  looks like this is used a few times in the tests